frontend/common/kubeflow-common-lib/projects/kubeflow/src/styles/styles.scss

```diff
@@ -5,6 +5,7 @@
 // Define the primary palette for the color that appears most
 // frequently throughout your application.
 $kubeflow-primary-palette: (
+  100: $kubeflow-blue-hue-1,
```

Comment:
I assume this is what was missing to make that TODO that I had put finally work.

Reply:
Yes it was just not there.